xds/src/main/java/io/grpc/xds/XdsServerBuilder.java

```diff
@@ -41,10 +46,17 @@
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/7514")
 public final class XdsServerBuilder extends ForwardingServerBuilder<XdsServerBuilder> {
 
+  public static final Attributes.Key<AtomicReference<FilterChainSelector>>
+          ATTR_FILTER_CHAIN_SELECTOR_REF = Attributes.Key.create(
```

Comment:
Would `InternalXdsAttributes` be the right place for defining this?

Reply:
From what are there in `InternalXdsAttributes `, seems so. Will move there. (I feel this hard to maintain tho, and more intuitive to put it to where it is created, also many other `internal` attributes still scattered around. )

Reply:
> ...I feel this hard to maintain tho, and more intuitive to put it to where it is created...

This is debatable but I can understand different people having different preferences. We should still follow one convention for where things are defined.